dspace-api/src/main/java/org/dspace/app/util/GoogleMetadata.java

```diff
@@ -180,9 +186,7 @@
             }
         }
 
-        if (log.isDebugEnabled()) {
-            logConfiguration();
-        }
+        logConfiguration();
```

Comment:
Normally I would not use #isDebugEnabled but, since the code is iterating over a HashMap and logging its content, the added efficiency is probably worthwhile.